src/plugins/intel_npu/src/compiler_adapter/include/ze_graph_ext_wrapper_interface.hpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ze_graph_ext.h>
+
+#include "intel_npu/network_metadata.hpp"
+
+namespace intel_npu {
+
+using SerializedIR = std::pair<size_t, std::shared_ptr<uint8_t>>;
+
+class ZeGraphExtWrappersInterface {
```

Comment:
Class name uses plural form "wrappers" while the file name uses "wrapper". Not sure this was intentional.

Reply:
done, thx